### ISSUE TITLE
Charging Tesla with PV divert in cold weather

### ISSUE THREAD

**Issue by Zuikkis** [OpenEVSE/ESP32_WiFi_V4.x#336]
Hi,

I'm from Finland.. It's the time of the year again when it's quite cold but solar plant is already producing over 6kW peaks.

Tesla charger starts up really slow in cold weather. I think it first heats the battery, and uses only battery power for that because the current draw will help heating. So when charging, OpenEVSE reports only 0.22A current (about 150W in 3 phase)! This can go on for 15-30 minutes or even longer if it's really cold. Then suddenly car decides it's good to go and starts charging.

It's usually no problem if you charge over night, but now when solar is available it's a bit bad. OpenEVSE first waits for 1400W (in 3 phase, so 4200W total) before it starts charging. But then car only takes 150W for next half an hour, and solar is wasted. What's worse, if there's clouds etc and solar is temporarily unavailable, OpenEVSE cuts power even is car is only taking 150W! 

Here's a graph from today, this is Domoticz logging the reported OpenEVSE power:

![kuva](https://user-images.githubusercontent.com/6397305/160391986-c446607e-1b39-4e0d-889d-993c6eb703d8.png)

See where it drops to zero a few times. I'm sure car would have started charging a lot earlier if there weren't those breaks?

Oh, I'm sorry to admit that I'm using 3.3.2 version. But I skimmed through later release notes and couldn't see anything related. Sorry if this is already changed!

TLDR: OpenEVSE shouldn't cut charging if car is not really drawing power! ;) Maybe ignore calculated amps if real amps are under 0.5A?

**Comment by glynhudson** [comment]
An easier workaround for this would be to increase the 'Minimum Charge Time` option in the solar PV divert advance settings
